arangod/Pregel/PregelFeature.cpp

```diff
@@ -274,15 +283,38 @@ ResultT<ExecutionNumber> PregelFeature::startExecution(TRI_vocbase_t& vocbase,
   }
 
   auto en = createExecutionNumber();
+
+  // TODO needs to be part of the conductor state
```

Comment:
Please create a ticket for this TODO

Reply:
this will also be part of the next actor-related PR